entrypoint.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-
 set -o errexit
 
+# init /srv if empty
+DIST_FILE=/srv/pmm-distribution
+if [ ! -f $DIST_FILE ]; then
+    echo "File $DIST_FILE doesn't exist. Initizlize /srv..."
+    echo docker > $DIST_FILE
+    mkdir -p /srv/{clickhouse,grafana,logs,postgres,prometheus,nginx,victoriametrics}
+    chown grafana:grafana /srv/grafana
+    chown pmm:pmm /srv/{victoriametrics,prometheus,logs}
+    chown postgres:postgres /srv/postgres
+    echo "Generate self-signed certificates for nginx"
+    bash /var/lib/cloud/scripts/per-boot/generate-ssl-certificate
```

Comment:
yeah, this could take some time. In air gaped env this will fail. I think we had PoC that we have copied it from /etc.